include/common/dm.h

```diff
@@ -17,11 +17,12 @@ enum {
 
 enum {
 	DM_CLASS_NONE = 0,
+	DM_CLASS_AXP_REGULATOR,
 	DM_CLASS_CLOCK,
 	DM_CLASS_IRQCHIP,
 	DM_CLASS_MSGBOX,
 	DM_CLASS_PIO,
-	DM_CLASS_REGULATOR,
+	DM_CLASS_SY_REGULATOR,
```

Comment:
This should not be in this commit (I know you are removing the other files, but they are in a self-contained commit).